src/main/java/com/shopify/model/ShopifyCustomCollection.java

```diff
@@ -0,0 +1,124 @@
+package com.shopify.model;
+
+import com.shopify.model.adapters.DateTimeAdapter;
+import org.joda.time.DateTime;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyCustomCollection {
```

Comment:
It looks like you are missing the published flag which is not read only.
You should add that flag to the custom collection model. From their docs, it looks like it will default to true, but this  seems trivial to be able to add the functionality to create an unpublished collection

https://help.shopify.com/en/api/reference/products/customcollection#create-2019-10